client/capiclient/clusters/clusters_client.go

```diff
@@ -0,0 +1,99 @@
+package clusters
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/giantswarm/apiextensions/pkg/clientset/versioned"
+	"github.com/giantswarm/gsclientgen/models"
+	"github.com/giantswarm/microerror"
+	"github.com/go-openapi/strfmt"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	clusterMaxAge = 1 // days
```

Comment:
But I dont get why you use it. What do you check here https://github.com/giantswarm/gsctl/pull/525/files#diff-7a40e503aeddc46df308598937fd015bR62 ?

AFAIK all clusters with deletion time are deleted or marked to be deleted

Reply:
I'm checking if the deleted clusters are older than one day. I agree that my implementation is kind of cryptic

The reason for doing it is that without the check, the command outputs clusters that have been deleted a long time ago (even January, even)
![image](https://user-images.githubusercontent.com/13508038/78545920-de1b0c80-77fc-11ea-8322-f378697e6eab.png)

I haven't seen this filter anywhere else, but the output without the check doesn't match with the current `gsctl` implementation. There seems to be something I'm missing... otherwise why it wouldn't return the same clusters?

Reply:
yeah I guess these CRs are leftovers CRs. Probably cluster service clean those out for the list action. Yeah I am with you we need to have a safeguard here. Maybe changing var naming to `clusterMaxValidAge` (not sure either, naming is hard 😅 )

Reply:
I've updated it, please let me know if it makes more sense this way